README.md

```diff
@@ -1,3 +1,9 @@
+[![Runbook](https://img.shields.io/badge/runbook-edit-172B4D.svg?logo=confluence)](https://dsdmoj.atlassian.net/wiki/spaces/laagetaccess/pages/1388150812/Applications+details)
+[![Exceptions](https://img.shields.io/badge/exceptions-view-FB4226.svg?logo=sentry)](https://sentry.service.dsd.io/mojds/prod-public/)
```

Comment:
🙋‍♂ I'm using the colours corresponding to the icon from https://simpleicons.org/. (The only exception is GitHub, which would be black...)